app/views/layouts/application.html.erb

```diff
@@ -5,12 +5,23 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+    <%= render partial: "shared/cdn_assets" %>
```

Comment:
Nice job adding your CDN assets to a partial, this helps keep your layout file clean!